server/methods/core/cart.js

```diff
@@ -511,8 +511,22 @@ Meteor.methods({
     if (cart.userId !== this.userId) {
       throw new Meteor.Error(403, "Access Denied");
     }
-    const order = Object.assign({}, cart);
+
+    let orders = [];
+    let order = Object.assign({}, cart);
+    delete order.items;
+    delete order.shopId;
     const sessionId = cart.sessionId;
+    const itemsByShop = _.groupBy(cart.items, "shopId");
```

Comment:
Order Splitting functionality such as splitting items by shop should probably be in a Separate PR to make these reviews go more smoothly.